infra/main.tf

```diff
@@ -38,18 +38,32 @@ module "lambda_add_pet" {
   lambda_role         = module.iam.iam_role_arn
   file_hash           = var.file_hash
   suffix              = "add_pet"
+  function_suffix     = "add_pet"
   s3_bucket_artifacts = var.s3_bucket_artifacts
   env_variables       = { s3_bucket_name : module.photo-s3bucket.s3_bucket_name }
 }
 
-module "lambda_statistics" {
+module "lambda_statistics_weekly" {
   source              = "./modules/lambda"
   app_name            = var.app_name
   dynamodb_table_name = module.dynamodb.dynamodb_table_name
   lambda_role         = module.iam.iam_role_arn
   file_hash           = var.file_hash
   suffix              = "statistics"
-  env_variables       = { s3_bucket_name : module.photo-s3bucket.s3_bucket_name }
+  function_suffix     = "weekly_statistics"
+  env_variables       = { s3_bucket_name : module.photo-s3bucket.s3_bucket_name, days = 7, email_title = "Pet of the days weekly statistics" }
+  s3_bucket_artifacts = var.s3_bucket_artifacts
+}
+
+module "lambda_statistics_monthly" {
+  source              = "./modules/lambda"
+  app_name            = var.app_name
+  dynamodb_table_name = module.dynamodb.dynamodb_table_name
+  lambda_role         = module.iam.iam_role_arn
+  file_hash           = var.file_hash
+  suffix              = "statistics"
+  function_suffix     = "monthly_statistics"
+  env_variables       = { s3_bucket_name : module.photo-s3bucket.s3_bucket_name, days = 30, email_title = "Pet of the days monthly statistics" }
```

Comment:
remove days variable -> we wanted monthly statistic to be overall not from last 30 days